### PR TITLE
Feature/movie info api

### PR DESCRIPTION
**Pull request by carolineshowscode**
## Is this a fix or a feature?
- feature

## What is the change?
- Changed MovieInfo back to a class component in `MovieInfo.js`
- Added fetch request for individual movie in `apiCalls.js`
- Added conditional rendering and componentDidMount to `MovieInfo.js`

## What does it fix?
- fetch timining fixed with rendering conditional

## Where should the reviewer start?
- MovieInfo.js

## How should this be tested?
- In browser

## What are the next steps?
- error handling
- home button
- user stories

### CODE REVIEW

File: src/components/MovieInfo.js [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+      }
+    }
+
+    componentDidMount = () => {
+      apiCalls.fetchAMovie(this.props.id)
+      .then(data => this.setState({movie: data.movie}))
+    }
+
+    render() {
+    if(!this.state.movie){
```
Comment (pcmueller) [inline src/components/MovieInfo.js:18]: Great call on the conditional here!

**Review by pcmueller — COMMENTED** [review]
Nicely done!!!

File: src/components/MovieInfo.js (hunk 1 above)
Comment (letakeane) [inline src/components/MovieInfo.js:18]: How could this be written in a way that uses the conventional conditional rendering syntax?

**Review by letakeane — COMMENTED** [review]
Lovely PR - great description, nice work. I like the focus on naming and the thoughtful use of lifecycle methods.

**Pull request merged**